design-proposals/passt-feature-gate-iface-api-deprecation.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+Following the emerging [Kubevirt Network Binding Plugin API](https://docs.google.com/document/d/13fLaqZuM8El_CT5XzxPv-1epKsB1-Spv_jtXveEkkdQ/edit#heading=h.z27rudfpwy44)
+and introduction of Passt network binding plugin [2](https://github.com/kubevirt/kubevirt/pull/10425) [3](https://github.com/kubevirt/kubevirt/pull/10621), 
+it's now possible to remove Passt network binding implementation from Kubevirt core.
+
+This doc outlines deprecation plan for Passt network binding, interface API and  feature gate.
```

Comment:
"This proposal outlines the replacement plan for the existing passt implementation & API."

The "deprecation" is an implementation step, it is preferred not to set it as the subject of this proposal.

Reply:
Done